test/bump-version-test/packages.dhall

```diff
@@ -0,0 +1,121 @@
+{-
```

Comment:
Do we need to version all of these? Why not do `spago init` before every test?

Reply:
This lot is versioned so that the package-set used by `packages.dhall` does not change. This means the `bower.json` in our test also does not need to change - so we can just do a simple comparison in the test assertion. If `spago init` is done every test run then the generated `bower.json` will be change when the versions in the package set change.

I am happy to change this if there is an alternative. I guess we could do a `spago init` and then edit `packages.dhall` to use a fixed package set URL... but that would make the test setup a bit more complicated.

I suppose, even keeping most stuff versioned, some stuff could be removed since it is not relevant - like source files and comments in `packages.dhall` ?

Reply:
> I guess we could do a spago init and then edit packages.dhall to use a fixed package set URL

Yeah I think this is the better thing to do even if it's slightly more involved, because the smaller amount of fixtures to update, the better (we'd need to update them because configuration changes schema, etc)

Reply:
Okay, I'll look at doing this.